VRCFaceTracking/Params/Eye/EyeTrackingParams.cs

```diff
@@ -94,16 +94,29 @@ public static class EyeTrackingParams
             new FloatEyeParameter(v2 =>
             {
                 if (v2.Left.Openness >= 1 && v2.Left.Widen > 0)
-                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Left.Widen); 
+                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Left.Widen);
                 return NormalizeFloat(0, 1, 0, 0.8f, v2.Left.Openness);
             }, "LeftEyeLidExpanded", true),
             new FloatEyeParameter(v2 =>
             {
                 if (v2.Right.Openness >= 1 && v2.Right.Widen > 0)
-                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Right.Widen); 
+                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Right.Widen);
                 return NormalizeFloat(0, 1, 0, 0.8f, v2.Right.Openness);
             }, "RightEyeLidExpanded", true),
 
+            new FloatEyeParameter(v2 =>
+            {
+                if (v2.Left.Openness >= 1 && v2.Left.Widen > 0)
+                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Left.Widen);
+                return v2.Left.Openness < .05 ? -v2.Left.Squeeze : NormalizeFloat(0, 1, 0, 0.8f, v2.Left.Openness);
+            }, "LeftEyeLidExpandedSqeeze", true),
+            new FloatEyeParameter(v2 =>
+            {
+                if (v2.Right.Openness >= 1 && v2.Right.Widen > 0)
+                    return NormalizeFloat(0, 1, 0.8f, 1, v2.Right.Widen);
+                return v2.Right.Openness < .05 ? -v2.Right.Squeeze : NormalizeFloat(0, 1, 0, 0.8f, v2.Right.Openness);
+            }, "RightEyeLidExpandedSqeeze", true),
```

Comment:
Small typo here! Should be 'Squeeze' for both LeftEyeLidExpandedSqeeze and RightEyeLidExpandedSqeeze.

Also I think you could probably check if the Squeeze value is greater than 0 instead of the Openness value being less than .05 since that might create a dead zone on the blinks (it's probably fine but would rather it be consistent with the other eyelid params).

Reply:
All the RightEyeLidExpandedSqueeze and LeftEyeLidExpandedSqueeze are with the "u"

Reply:
fixed 15 days later